pkg/brokerapi/service_instance.go

```diff
@@ -31,16 +31,9 @@ type ServiceInstance struct {
 	Parameters map[string]interface{} `json:"parameters, omitempty"`
 }
 
-// LastOperation represents the state of a discrete action that the broker is
-// completing asynchronously
-type LastOperation struct {
-	State       string `json:"state"`
-	Description string `json:"description,omitempty"`
-}
-
-// ServiceInstanceRequest represents a request to a broker to provision an
+// CreateServiceInstanceRequest represents a request to a broker to provision an
 // instance of a service
-type ServiceInstanceRequest struct {
+type CreateServiceInstanceRequest struct {
```

Comment:
what's the reason for renaming this particular struct? it's creating a lot of spurious changes for review.